Sources/COWMacros/SyntaxExtensions.swift

```diff
@@ -129,6 +137,107 @@ extension FunctionDeclSyntax {
         }
       }
     }
+    return false
+  }
+  
+  var isMutating: Bool {
+    modifiers.contains(where: { $0.name.tokenKind == .keyword(.`mutating`) })
+  }
+  
+  func returnTypeEquals(to type: String) -> Bool {
+    guard let returnClause = signature.returnClause,
+          let returnType = returnClause.type.as(IdentifierTypeSyntax.self) else {
+      return type == "Void"
+    }
+    return returnType.name.trimmed.text == type
+  }
+  
+  // Equatable:
+  // static func == (lhs: Self, rhs: Self) -> Bool
+  func likelyToConformToEquatable(for structDecl: StructDeclSyntax) -> Bool {
```

Comment:
Explicit `internal` access control is recommended. This convention is learned from the Swift Standard Library.